man/systemd-system.conf.xml

```diff
@@ -162,6 +162,21 @@
         These settings have no effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>PretimeoutWatchdogSec=</varname></term>
```

Comment:
"Pretimeout" → "PreTimeout", no?

i.e. "pretimeout" sounds weird to me, I'd always write "pre-timeout". But then again I am not a native english speaker...

Reply:
still not updated